stistools/ocrreject.py

```diff
@@ -323,15 +318,15 @@ def ocrreject(input, output,
                 arglist.append("-t")
             arglist.extend(optional_args)
 
-        if verbose:
-            print("Running ocrreject on {}".format(infile))
-            print("  {}".format(str(arglist)))
-        status = subprocess.call(arglist, stdout=fd_trailer,
-                                 stderr=subprocess.STDOUT)
-        if status:
-            cumulative_status = 1
             if verbose:
```

Comment:
@stscirij -

Found a logic bug due to the indentation here.  It should now process multiple files when `all=False`.